mod/ic/soic-24-w.yaml

```diff
@@ -0,0 +1,14 @@
+chip_shape: [7.5, 15.4]
+description: SOIC-24-W
+model:
+  offset: [0, 0, 0]
+  path: ${KISYS3DMOD}/Package_SO.3dshapes/SOIC-24W_7.5x15.4mm_P1.27mm.wrl
+  rotate: [0, 0, 0]
+  scale: [1, 1, 1]
+name: SOIC-24-W
+pad_shape: [2.20, 0.60]
```

Comment:
Datasheet for A3967 has 0.65 wide pads, but 0.60 is probably fine too, was this deliberate?

Reply:
No this was not!